gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -219,6 +216,10 @@ def add_arguments(self, parser):
         help=('Value to use instead of null for numeric (float/int/long) lists.'
               'For instance, [0, None, 1] will become '
               '[0, `null_numeric_value_replacement`, 1].'))
+    parser.add_argument(
+        '--include_call_name',
+        type='bool', default=False, nargs='?', const=True,
+        help=('Add raw sample name as a separate column in sample record.'))
```

Comment:
how about:
''Add raw sample name (from VCF Header line) as a subfield under the call column in addition to sample_id.''

Reply:
Done.